soepy/test/random_init.py

```diff
@@ -214,6 +217,15 @@ def random_init(constr=None):
     )
     exog_child_info.to_pickle("test.soepy.child.pkl")
 
+    # Generate random probabilities of marriage
+    index_levels = [list(range(0, periods)), [0, 1, 2]]
+
+    index = pd.MultiIndex.from_product(index_levels, names=["period", "educ_level"])
```

Comment:
See comments to the creation of this frame at the very beginning.

Reply:
Comment above applies